stock_reserve_sale/README.rst

```diff
@@ -28,15 +30,24 @@ will get a message suggesting to reserve each line individually. There is no
 module dependency: this modules is fully functional even without ownership
```

Comment:
Is this a good idea?
What if you need the module ownership for legacy reason reason (or any other module depending on it)?